view/frontend/web/template/payment/checkout-bypass.html

```diff
@@ -10,7 +10,7 @@
             <div class="checkout-logo">
                 <div class="checkout-logo-container">
                     <img src="https://cdn2.hubspot.net/hubfs/2610868/Logot/op-logo-30x30px.png"
```

Comment:
I just remembered that we have to update the logo as well! 
I'll upload new logo to HS CDN and update the link before we merge this